### PR TITLE
Update power-plant.yaml

### PR DESCRIPTION
**Pull request by sandrinecharousset**
adding changes from closed PR 
add Variable Cost (incl. Fuel Cost)
add seasonality
remove initial storage
remove number of units

### CODE REVIEW

File: definitions/variable/technology/power-plant.yaml [hunk 1]
```diff
@@ -199,26 +192,19 @@
...
-    - Monthly
-- Stored Energy Inventory|Electricity|Energy Storage System:
-    description: State of the stored energy by ESS
-    unit:
-    - MWh
-    - GWh
+    unit: GWh
+- Seasonality|Electricity|Energy Storage System|{Energy Storage System Type}:
+    description: seasonality (daily, weekly, monthly, etc.) of {Energy Storage System Type}
+    unit: [days,weeks,months]
```
Comment (erikfilias) [inline definitions/variable/technology/power-plant.yaml:204]:
```suggestion
    unit: [daily,weekly,monthly]
```

File: definitions/variable/technology/power-plant.yaml [hunk 2]
```diff
@@ -199,26 +192,19 @@
...
+- Seasonality|Electricity|Hydro|Reservoir:
+    description: seasonality (daily, weekly, monthly, etc.) of Reservoir
+    unit: [days,weeks,months]
```
Comment (erikfilias) [inline definitions/variable/technology/power-plant.yaml:207]:
```suggestion
    unit: [daily,weekly,monthly]
```

File: definitions/variable/technology/power-plant.yaml [hunk 3]
```diff
@@ -199,26 +192,19 @@
...
+- Seasonality|Electricity|Hydro|Pumped Storage:
+    description: seasonality (daily, weekly, monthly, etc.) of Pumped Storage
+    unit: [days,weeks,months]
```
Comment (erikfilias) [inline definitions/variable/technology/power-plant.yaml:210]:
```suggestion
    unit: [daily,weekly,monthly]
```

**Review by erikfilias — COMMENTED** [review]
Regarding the seasonality, I would rather be consequent and use daily, weekly, or monthly. Since when it mentions "days", it could refer to weekly management.

**Comment by sandrinecharousset** [comment]
> Regarding the seasonality, I would rather be consequent and use daily, weekly, or monthly. Since when it mentions "days", it could refer to weekly management.

I approve the changes as it is your variable but I am a bit doubtful… daily does not sound as a unit… what will be the values associated?

**Review by erikfilias — APPROVED** [review]

**Comment by sandrinecharousset** [comment]
@danielhuppmann I think this PR can be merged now?

File: definitions/variable/technology/power-plant.yaml [hunk 4]
```diff
@@ -199,26 +192,19 @@
...
-    - Monthly
-- Stored Energy Inventory|Electricity|Energy Storage System:
-    description: State of the stored energy by ESS
-    unit:
-    - MWh
-    - GWh
+    unit: GWh
+- Seasonality|Electricity|Energy Storage System|{Energy Storage System Type}:
+    description: seasonality (daily, weekly, monthly, etc.) of {Energy Storage System Type}
+    unit: [daily,weekly,monthly]
```
Comment (danielhuppmann) [inline definitions/variable/technology/power-plant.yaml:204]: This is not really a unit, I guess, but the allowed values? Non-numeric values are not foreseen in the IAMC timeseries data format.

**Review by danielhuppmann — APPROVED** [review]
I will merge for now, but we will need to discuss the unit and allowed values of the Seasonality-variables...

**Pull request merged**